orm/internal/codegen/file.go

```diff
@@ -0,0 +1,154 @@
+package codegen
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/cosmos/cosmos-proto/generator"
+	"github.com/iancoleman/strcase"
+	"google.golang.org/protobuf/compiler/protogen"
+	"google.golang.org/protobuf/proto"
+
+	v1alpha1 "github.com/cosmos/cosmos-sdk/api/cosmos/orm/v1alpha1"
+)
+
+var (
+	tablePkg = protogen.GoImportPath("github.com/cosmos/cosmos-sdk/orm/model/ormtable")
+)
+
+type fileGen struct {
+	*generator.GeneratedFile
+	file *protogen.File
+}
+
+func (f fileGen) gen() error {
+	f.P("// Code generated by protoc-gen-go-cosmos-orm. DO NOT EDIT.")
+	f.P()
+	f.P("package ", f.file.GoPackageName)
+	stores := make([]*protogen.Message, 0)
+	for _, msg := range f.file.Messages {
+		tableDesc := proto.GetExtension(msg.Desc.Options(), v1alpha1.E_Table).(*v1alpha1.TableDescriptor)
+		if tableDesc != nil {
+			newTableGen(f, msg, tableDesc).gen()
+		}
+		singletonDesc := proto.GetExtension(msg.Desc.Options(), v1alpha1.E_Singleton).(*v1alpha1.SingletonDescriptor)
+		if singletonDesc != nil {
+			// do some singleton magic
+			newSingletonGen(f, msg, singletonDesc).gen()
+		}
+
+		if tableDesc != nil || singletonDesc != nil { // message is one of the tables,
+			stores = append(stores, msg) // TODO(Tyler): maybe we should split this into singleton stores vs table stores
+		}
+	}
+	f.genStoreInterfaces(stores)
+	f.genStoreStruct(stores)
+	f.genStoreGetters(stores)
+	f.genStoreInterfaceGuard()
+	f.genStoreConstructor(stores)
+	return nil
+}
+
+func (f fileGen) genStoreInterfaces(stores []*protogen.Message) {
+	f.P("type ", f.storeInterfaceName(), " interface {")
+	for _, store := range stores {
+		f.P(store.GoIdent.GoName, "()", f.messageStoreInterfaceName(store))
```

Comment:
```suggestion
		f.P(f.messageStoreInterfaceName(store), "()", f.messageStoreInterfaceName(store))
```

so we get `BalanceStore()` instead of `Balance()`